### PR TITLE
Pollitzd huntsville

### PR DESCRIPTION
**Pull request by pollitzd**
- This is a bullet.  CRAZY.

1. This is a numbered list. EVEN CRAZIER.

- [ ] This is a task list.  THE CRAZIEST OF ALL.

### CODE REVIEW

**Comment by dacrouch** [comment]
this has no unit tests

File: pollitzd-picayune.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+### Places to Eat
+- Gator Me Crazy
+- Taco Bell 
+
+
+
+### Places to Visit
+- Crosby Arboretum
+- bananas stands
```
Comment (dacrouch) [inline pollitzd-picayune.md:12]: not enough bananas

Comment (pollitzd) [reply pollitzd-picayune.md:12]: There are plenty of bananas.

Comment (pollitzd) [reply pollitzd-picayune.md:12]: HAPPY????

**Review by dacrouch — APPROVED** [review]
This does not really meet standards, but I guess we'll have to go with it.

File: pollitzd-picayune.md (hunk 1 above)
Comment (dacrouch) [reply pollitzd-picayune.md:12]: :-1:

**Pull request merged**